### DISCUSSION TITLE
Capitalized model names and writing raw SQL

### DISCUSSION THREAD

**Discussion by bdelamotte** [prisma/prisma#10357] [Q&A] [answered]
Hello,

I have all my models in schema.prisma capitalized such as 
``` 
model Session {...}
model User {...}
model Account {...}
```

However, when I have to connect to the database to write some ad-hoc queries now everything requires me double quoting the tables in order to use them. Is there a way to tell Prisma that when creating these tables, to not use double quotes so they default to lowercase. 

The problem is I keep having to write my SQL like this:
```
Select * from "Session"
```
Instead of this:
```
select * from session
```

The problem happens with column names too.

If not, is there a way to tell the database, to not require things to be capitalized to find the table?

Thanks!

**Comment by janpio** [comment, marked as answer]
You can probably use `@@map` on the models to tell it use a different name in the database (`session` vs. `Session`): https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#mapping-model-names-to-tables-or-collections

**Reply by bdelamotte** [reply]
Thanks for the link. This would still be very tedious for every table and column in the schema.prisma file to do. 

TypeORM has something cool that I hope Prisma could do someday. https://www.npmjs.com/package/typeorm-naming-strategies

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
